jenkins/check-for-build.jenkinsfile

```diff
@@ -25,6 +25,8 @@ pipeline {
         parameterizedCron '''
             H 1 * * * %INPUT_MANIFEST=1.3.15/opensearch-dashboards-1.3.15.yml;TARGET_JOB_NAME=distribution-build-opensearch-dashboards;BUILD_PLATFORM=linux windows;BUILD_DISTRIBUTION=tar rpm deb zip
             H 1 * * * %INPUT_MANIFEST=1.3.15/opensearch-1.3.15.yml;TARGET_JOB_NAME=distribution-build-opensearch;BUILD_PLATFORM=linux macos windows;BUILD_DISTRIBUTION=tar rpm deb zip
+            H */1 * * * %INPUT_MANIFEST=2.13.0/opensearch-dashboards-2.13.0.yml;TEST_MANIFEST=2.13.0/opensearch-dashboards-2.13.0-test.yml;TARGET_JOB_NAME=distribution-build-opensearch-dashboards;BUILD_PLATFORM=linux windows;BUILD_DISTRIBUTION=tar rpm deb zip
```

Comment:
```suggestion
            H */1 * * * %INPUT_MANIFEST=2.13.0/opensearch-dashboards-2.13.0.yml;TARGET_JOB_NAME=distribution-build-opensearch-dashboards;BUILD_PLATFORM=linux windows;BUILD_DISTRIBUTION=tar rpm deb zip
```

Reply:
@rishabh6788 this is our moving target (development brach ) - we would expect many changes there

Reply:
I would recommend it to run every 6 hours or so since our Jenkins CI system is over utilized at the moment due to 2.12.0 RC build / test process.